pool-manager.Dockerfile

```diff
@@ -0,0 +1,54 @@
+#
+# This Dockerfile builds a recent cspi-mgmt using the latest binary from
+# cspi-mgmt  releases.
+#
+
+#openebs/cstor-base is the image that contains cstor related binaries and
+#libraries - zpool, zfs, zrepl
+#FROM openebs/cstor-base:ci
+ARG BASE_IMAGE
```

Comment:
Not a fan of using an ARG for `BASE_IMAGE` if it's a 1:1 variable we substitute anyways.